opencensus/stats/exporters/prometheus_exporter.py

```diff
@@ -345,4 +347,14 @@ def get_view_name(namespace, view):
     name = ""
     if namespace != "":
         name = namespace + "_"
-    return name + view.name
+    return sanitize(name + view.name)
+
+
+_PROMETHEUS_FORMAT_RE = re.compile('[^a-zA-Z0-9_]')
```

Comment:
The more appropriate/correct and concise regex would perhaps be:
  `re.compile('[^\w]', re.UNICODE|re.IGNORECASE)`

Reply:
And let's call it, `_NON_LETTERS_NOR_DIGITS_RE`

Reply:
Neat - updated. Thanks for suggesting on Python code style!